RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1801,6 +1801,10 @@ boolean L11_shenStartQuest() {
 	{
 		return false;
 	}
+	if (my_daycount() < 2 || !allowSoftblockShen()) {
+		// if you're fast enough to open it on day 1, maybe wait until day 2
```

Comment:
I wanted to say "but what if you run out of things to do entirely"... but then I realized we would probably never be able to do 1 day runs in autoscend.

Reply:
It's likely possible in unrestricted with like every Mr Store & KoLcon item from the last 15 years but we can deal with that problem if it ever comes up.
I think people are scripting 1 day HCCS these days but that's more of an optimisation problem than anything else.